resources/validate_builds.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+# validated if created build is debug build 
+cd $APK_PATH
+EXPECTED_FLAG='application-debuggable'
+
+for file in *
+do 
+  name=$(echo $file| tail -c 4)
+  if [ $name = 'apk' ]
```

Comment:
I'm still a little confused by this part. Based upon your earlier comment 'we may expect multiple apks', then this loop basically just grabs the first .apk file in the directory. But, if there can be more than one apk, does it matter which one you choose? I think it would be better to be explicit here. So, if there are multiple apks generated, but there is one specific one you care about, why not hard-code it?